pkgs/tools/system/htop/default.nix

```diff
@@ -32,6 +32,11 @@ stdenv.mkDerivation rec {
     ++ optional sensorsSupport "--with-sensors"
   ;
 
+  postInstall = ''
+    # link $prefix/etc to /etc (sysconfdir is $prefix/etc) to fix reading global htoprc
+    ln -s /etc $out/etc
+  '';
```

Comment:
Can we adjust the path in htop instead? such symlinks are ugly.

Reply:
Not sure how I should set sysconfdir, seems to be `$(prefix)/etc` and prefix is basically where it should put stuff, so can't change that

Reply:
That should be referenced in code somewhere and we can hijack the path there.